README.md

```diff
@@ -285,6 +285,7 @@ cirros | Cirros
 fedora | Fedora (amd64)
 fedora.arm64 | Fedora (arm64)
 fedora.s390x | Fedora (s390x)
+generic | Generic Linux Guest
```

Comment:
`generic` suggests this could be anything so I'd rather just call this `linux`.

Reply:
Done.